Source/OIDServiceConfiguration.m

```diff
@@ -54,25 +59,27 @@ @implementation OIDServiceConfiguration
 
 @synthesize authorizationEndpoint = _authorizationEndpoint;
 @synthesize tokenEndpoint = _tokenEndpoint;
+@synthesize issuer = _issuer;
 @synthesize registrationEndpoint = _registrationEndpoint;
 @synthesize discoveryDocument = _discoveryDocument;
 
 - (instancetype)init
     OID_UNAVAILABLE_USE_INITIALIZER(@selector(
         initWithAuthorizationEndpoint:
-                       tokenEndpoint:
-                registrationEndpoint:)
+                       tokenEndpoint:)
     );
 
 - (instancetype)initWithAuthorizationEndpoint:(NSURL *)authorizationEndpoint
```

Comment:
Looks like this is the actual designated initializer? Should we mark it as such in the header?

Reply:
So this method is not actually exposed in the header at all.

We're expecting people to either specify the endpoints manually, OR use a discovery doc – but not both. Do you think we should revise this?